### PR TITLE
Detect shortname change

### PR DESCRIPTION
**Pull request by deniak**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -50,6 +50,23 @@ app.use(express.static("public"));
...
+        ,   handler = new Sink;
+        v.extractMetadata({url: url, events: handler});
+        handler.on("exception", function (data) {
+            res.status(500).json(data);
+        });
+        handler.on("end-all", function () {
+            res.status(200).json(v.meta);
+        });
+    }
+});
```
Comment (tripu) [inline app.js]: Revert these changes?

File: lib/rules/metadata/dl.js [hunk 2]
```diff
@@ -7,23 +7,33 @@ exports.name = 'metadata.dl';
...
         result['previousVersion'] = $linkPrev.attr('href').trim();
+        previousShortname = $linkPrev.attr("href").trim().match(new RegExp(/.*\/[^/-]+\-(.*)-\d{8}\/$/))[1];
+    }
 
     var $linkEd = (dts.Editor) ? dts.Editor.dd.find("a").first() : null;
     if ($linkEd && $linkEd.length)
         result['editorsDraft'] = $linkEd.attr('href').trim();
 
+    if (previousShortname && latestShortname && previousShortname !== latestShortname) {
+        result['sameWorkAs'] = 'https://www.w3.org/TR/'+previousShortname+'/';  // TODO use api to get the previous shortlink
```
Comment (tripu) [inline lib/rules/metadata/dl.js]: I don't understand the `TODO` here, @deniak.

**Comment by deniak** [comment]
@tripu ready for your review. The `TODO` is there because we are building the `sameWorkAs` url instead of retrieving it via the API. This will avoid updating the rule again after we switch to `https` for instance.
